### PR TITLE
Rename deposits to payouts on the settings screen

### LINKED ISSUE #9527: Payouts Rename: Settings Page
### Description

Rename `Deposits` to `Payouts` on the Payment settings page.

### PR DESCRIPTION
**Pull request by brucealdridge**
Fixes #9527 

#### Changes proposed in this Pull Request

Rename "Deposits" to "payouts on the settings screen and notices.

This PR updates all instances of "Deposits" to "payouts" on the settings screen

Screenshot 2024-10-09 at 10 38 30 AM

and all the "deposit" notices.

image

#### Testing instructions

* Visit Payments > Settings
* Check the UI, including changing the deposit schedule.
* Test various error states.

Rather than testing various error states, I injected all deposit notices onto a page to review.

```
<hr/>

 <h3>Notices Tests</h3>
 
  <DepositFailureNotice updateAccountLink={'#'}/>
  <SuspendedDepositNotice/>
  <DepositIncludesLoanPayoutNotice/>
  <NewAccountWaitingPeriodNotice/>
  <DepositTransitDaysNotice/>
  <NegativeBalanceDepositsPausedNotice/>
  <DepositMinimumBalanceNotice minimumDepositAmountFormatted={'77.32'}/>
  <NoFundsAvailableForDepositNotice/>

<hr/>
```

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9532` or branch name `fix/9527-payouts-rename-settings` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix/9527-payouts-rename-settings) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: de6bd7976f74190161a07d1213bdc8985435760c
- Build time: 2024-10-21 19:08:14 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/components/deposits-overview/deposit-notices.tsx
Comment (haszari) [inline client/components/deposits-overview/deposit-notices.tsx:200]: Need to review this in context to confirm the new wording works, and/or get design or product feedback.

File: client/settings/deposits/index.js
Comment (haszari) [inline client/settings/deposits/index.js]: This might need tweaking, second payouts seems redundant. 

Would this communicate everything we need to say here?

> Manage and update your payout account information to receive payments.

File: client/settings/settings-manager/index.js
Comment (haszari) [inline client/settings/settings-manager/index.js]: Might be worth reviewing wording here, what are we trying to say? Maybe something like…

> Funds are dispatched to your payout account N business days after they're received.

or

> Funds are available for payout N business days after they're received.

File: client/settings/deposits/index.js
Comment (haszari) [reply client/settings/deposits/index.js]: Or maybe:

> Manage and update your bank account information to receive payouts.

**Review by haszari — APPROVED** [review]
Code changes look good, I didn't see any missed "deposit" in settings screen. 

I didn't exhaustively test, here's what I tested:

- Payments > Settings screen payouts box and title/heading
- 3x of the updated notices:
  - new account waiting period in settings screen
  - new account waiting period in overview screen
  - 1-3 biz days warning in overview screen

#### Next steps
- This can't merge yet, we need to work out what we're doing. 
  - Maybe a feature branch?
- Some wording may need a review from design and/or product. Not a blocker but might be able to simplify/improve while we're here.
- Would be good to test more of the notices/changes in context. Will also help refine wording.

Perhaps we could merge these PRs to a feature branch. Then we can do a testing / UX review blitz with all changes in context.

I also noted all the relevant docs links. I'm thinking we'd need to ship in this order:

1. Get the changes ready so we have most (~80% or more of UI) updated to `payout`, ready to deploy in a specific WooPayments release + service PR. (i.e. 2x feature branches, one per repo)
2. Update the docs AND URLs first, on live docs site. All pages will have redirects so the old URLs still work (e.g. `http://../deposit-schedules`).
3. Ship/deploy: 
  - Deploy server PR (e.g. emails, etc).
  - Ship plugin with release note for most/80%+ changes.

This way, there will be a brief period (~hours/day or two max) when the docs say `payout` but the software says `deposit`, but links will always work and nothing will be broken, and the change will happen mostly "atomically" (i.e. the docs links in product will ship with the UI changes).

That's just my thinking, keen to discuss as a team internally before we proceed.

File: client/components/deposits-overview/deposit-notices.tsx
Comment (shendy-a8c) [reply client/components/deposits-overview/deposit-notices.tsx:200]: I confirm that the removal of those words `to deposit` still makes sense since `<NoFundsAvailableForDepositNotice />` is currently only displayed on the overview page within the Deposit section ([ref](https://github.com/Automattic/woocommerce-payments/blob/d7861f8df73a7b5666d07e653c6c588199db8309/client/components/deposits-overview/index.tsx#L164)).

Screenshot 2024-10-22 at 01 53 26

File: client/settings/deposits/index.js
Comment (shendy-a8c) [reply client/settings/deposits/index.js]: I went ahead with

> Manage and update your bank account information to receive payouts.

[30ac371](https://github.com/Automattic/woocommerce-payments/pull/9532/commits/30ac37106c2d8723d58e2f62e6a4367fbbda733f)

File: client/settings/settings-manager/index.js
Comment (shendy-a8c) [reply client/settings/settings-manager/index.js]: I went ahead with

> Funds are available for payout N business days after they're received.

[de6bd79](https://github.com/Automattic/woocommerce-payments/pull/9532/commits/de6bd7976f74190161a07d1213bdc8985435760c)

**Review by shendy-a8c — APPROVED** [review]
- [x] Reviewed code changes and they look good.
- [x] Tested with local changes that display all the notices.
- [x] Addressed all the feedbacks.

**Pull request merged**
